File: weather-mvp-dagger/src/main/java/com/baronzhang/android/weather/widget/IndicatorValueChangeListener.java
package com.baronzhang.android.weather.widget;

/**
 * @author baronzhang (baron[dot]zhanglei[at]gmail[dot]com)
 */
public interface IndicatorValueChangeListener {

    void onChange(int currentIndicatorValue, String stateDescription, int indicatorTextColor);
}

File: weather-mvp-dagger-kotlin/src/main/java/com/minimalist/weather/kotlin/widget/IndicatorValueChangeListener.kt
package com.minimalist.weather.kotlin.widget

/**
 * @author baronzhang (baron[dot]zhanglei[at]gmail[dot]com)
 */
interface IndicatorValueChangeListener {

    fun onChange(currentIndicatorValue: Int, stateDescription: String, indicatorTextColor: Int)
}
